WalnutExternal.lua

```diff
@@ -3,20 +3,62 @@
 VULKAN_SDK = os.getenv("VULKAN_SDK")
 
 IncludeDir = {}
-IncludeDir["VulkanSDK"] = "%{VULKAN_SDK}/Include"
+IncludeDir["VulkanSDK"] = "%{VULKAN_SDK}/include"
 IncludeDir["glm"] = "../vendor/glm"
 
 LibraryDir = {}
-LibraryDir["VulkanSDK"] = "%{VULKAN_SDK}/Lib"
+LibraryDir["VulkanSDK"] = "%{VULKAN_SDK}/lib"
 
 Library = {}
-Library["Vulkan"] = "%{LibraryDir.VulkanSDK}/vulkan-1.lib"
+
+if os.ishost("linux") and os.istarget("linux") then
+   Library["Vulkan"] = "%{LibraryDir.VulkanSDK}/libvulkan.so.1"
+else
+   Library["Vulkan"] = "%{LibraryDir.VulkanSDK}/vulkan-1.lib"
+end 
+
+function linux_check_if_package_is_installed(cmd, package)
+   local result, errorCode = os.outputof(cmd .. " " .. package)
+   if errorCode ~= 0 then
+      premake.error("Install " .. package .. " using package manager")
+   end
+   premake.info(package .. " is installed")
+end
 
 group "Dependencies"
    include "vendor/imgui"
-   include "vendor/glfw"
+   if os.ishost("linux") and os.istarget("linux") then
```

Comment:
The filter keyword seems to be best practice for setting OS specific settings.
`filter "system:linux"`